_posts/1970-01-01-wmbattery.md

```diff
@@ -26,14 +26,17 @@ versions:
   number: 2.49
   download: wmbattery-2.49.tar.gz
  -
-  number: "2.50"
+  number: 2.50
```

Comment:
These quotes make sure that Jekyll prints the trailing zero, i.e., 2.50 instead of 2.5.  Let's keep them.

Reply:
Ah, right, I did wonder. Will fix.

Reply:
Amended and force-pushed.